### PR TITLE
New verson: Cisco.Webex.Bundle verson 44.12.0.31404

### LINKED ISSUE #199788: [Package Request]: Cisco Webex App and desktop meetings bundle
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://help.webex.com/en-us/article/nw5p67g/Webex-App-%7C-Installation-and-automatic-upgrade
Publisher: Cisco Systems, Inc
Package Name: Cisco.Webex.Bundle
Description: Cisco Webex is an app for continuous teamwork. Move work forward in secure work spaces where everyone can contribute anytime with messaging, file sharing, white boarding, video meetings, calling, and more. It works on virtually any device, with these top benefits for mobile app users:
  - Streamline Teamwork: One place for all the tools that help you do your job better and faster. Create spaces for you and another person or for a group working on a project.
  - Make Better Decisions: Start and join meetings instantly. See everyone with an amazing HD video experience. Share your screen. Keep teamwork going in the connected work space.
  - Stay Informed About All Your Work: Select from our growing catalog of integrations and bots to keep up to date with what's happening in other apps such as Box and Salesforce
  - Find Anything Fast: Search for people, messages, and files across all your spaces - no need to switch accounts or views
  - Bring The Experience Into Meeting Rooms: Pair your app to a Webex device, such as the Webex Board, to do things like wirelessly control meetings and save whiteboard drawings
  - Work Securely: Built for business collaboration with end-to-end encryption
Package Version: 44.12.0.31404
Installer Url: https://binaries.webex.com/Webex-Desktop-Windows-x64-Combined-Gold/WebexBundle.msi

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #199788

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

image

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199835)

### CODE REVIEW

File: manifests/c/Cisco/Webex/Bundle/44.12.0.31404/Cisco.Webex.Bundle.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- DisplayName: Webex
+  Publisher: Cisco Systems, Inc
+  ProductCode: '{A04D0B8C-CF5F-5B72-8BD5-8A899DB199EE}'
+  UpgradeCode: '{B9DCA8E9-B3A9-419A-9D6F-9BC3557EE72C}'
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\Cisco Spark'
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: http://asus-gt302/d/WebexBundle.msi
```
Comment (Dragon1573) [inline manifests/c/Cisco/Webex/Bundle/44.12.0.31404/Cisco.Webex.Bundle.installer.yaml:36]:
```suggestion
  InstallerUrl: https://binaries.webex.com/Webex-Desktop-Windows-x64-Combined-Gold/WebexBundle.msi
```

Uses a local server as installer file cache.

Accidentally committed without switching back to the official URL. 🤣

**Comment by stephengillie** [comment]
URL: https://binaries.webex.com/Webex-Desktop-Windows-x64-Combined-Gold/WebexBundle.msi 
Status Code: 200

(Automated message - build 931)

**Comment by stephengillie** [comment]
image
Is the package available from another host?

File: manifests/c/Cisco/Webex/Bundle/44.12.0.31404/Cisco.Webex.Bundle.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\Cisco Spark'
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://binaries.webex.com/Webex-Desktop-Windows-x64-Combined-Gold/WebexBundle.msi
+  InstallerSha256: E450F6CDC89649165FA746785457165B09CC63773B34B528E541433AA0E02FCF
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: http://asus-gt302/d/WebexBundle.msi
```
Comment (Dragon1573) [inline manifests/c/Cisco/Webex/Bundle/44.12.0.31404/Cisco.Webex.Bundle.installer.yaml:40]:
```suggestion
  InstallerUrl: https://binaries.webex.com/Webex-Desktop-Windows-x64-Combined-Gold/WebexBundle.msi
```

**Comment by Dragon1573** [comment]
Now should be fixed.

Forget to switch back to the official download URL from my local "cache server" almost every time ...

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**